resources/crds/ripsaw_v1alpha1_postgres_cr.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: ripsaw.cloudbulldozer.io/v1alpha1
+kind: Benchmark
+metadata:
+  name: postgres-infra
+  namespace: ripsaw
+spec:
+  infrastructure:
+    name: postgres
+    args:
+      servers:
+        # Typical deployment size is 3
+        size: 1
+      storage:
+        # persistent storage volume is required
```

Comment:
In the docs you mention, `use_persistent_storage: True` however here it mentions persistent storage is required. Do we require it?

Reply:
Required persistent storage was a later discovery about the operator. I'll update the doc to match.